test/test_support.ts

```diff
@@ -279,56 +243,63 @@ function extractInlineSourceMap(source: string): string {
     result = inlineSourceMapRegex.exec(source);
   } while (result !== null);
   const base64EncodedMap = previousResult![1];
-  return Buffer.from(base64EncodedMap, 'base64').toString('utf8');
+  const sourceMapJson = Buffer.from(base64EncodedMap, 'base64').toString('utf8');
+  return sourceMapTextToConsumer(sourceMapJson);
 }
 
-function getFileWithName(filename: string, files: Map<string, string>): string|undefined {
-  for (const filepath of toArray(files.keys())) {
+export function getFileWithName(filename: string, files: Map<string, string>): string {
+  const filePaths = toArray(files.keys());
+  let file = '';
+  for (const filepath of filePaths) {
```

Comment:
Just tried it and apparently we don't have access to iterators in tsickle.  Do you know why we're not on (I assume) a higher language level?